routes/web.php

```diff
@@ -12,7 +14,8 @@
 | contains the "web" middleware group. Now create something great!
 |
 */
-
-Route::get('/', function () {
-    return view('welcome');
-});
+////utana
```

Comment:
it should be erased than make a doubt in this field.try to show which is important